scripts/main.js

```diff
@@ -0,0 +1,241 @@
+const num = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+let disChar = document.querySelector('.characterdisplay');
+
+const disResult = document.querySelector('.resultdisplay');
+
+const btnseven = document.querySelector('.seven');
+btnseven.addEventListener('click', seven);
+
+function seven() {
+  disChar.innerHTML += num[7];
+  return num[7];
+}
+
+const btneight = document.querySelector('.eight');
+btneight.addEventListener('click', eight)
+
+function eight() {
+  disChar.innerHTML += num[8]
+  return num[8];
+}
+
+const btnnine = document.getElementById('nine');
+btnnine.addEventListener('click', nine);
+
+function nine() {
+  disChar.innerHTML += num[9];
+  return num[9];
+}
+
+const btnfour = document.getElementById('four');
+btnfour.addEventListener('click', four);
+
+function four() {
+  disChar.innerHTML += num[4];
+  return num[4];
+}
+
+const btnfive = document.getElementById('five');
+btnfive.addEventListener('click', five);
+
+function five() {
+  disChar.innerHTML += num[5];
+  return num[5];
+}
+
+const btnsix = document.getElementById('six');
+btnsix.addEventListener('click', six);
+
+function six() {
+  disChar.innerHTML += num[6];
+  return num[6];
+}
+
+const btnone = document.getElementById('one');
+btnone.addEventListener('click', one);
+
+function one() {
+  disChar.innerHTML += num[1];
+  return num[1];
+}
+
+const btntwo = document.getElementById('two');
+btntwo.addEventListener('click', two);
+
+function two() {
+  disChar.innerHTML += num[2];
+  return num[2];
+}
+
+const btnthree = document.getElementById('three');
+btnthree.addEventListener('click', three);
+
+function three() {
+  disChar.innerHTML += num[3];
+  return num[3];
+  console.log(num[3])
+}
+
+const btnzero = document.getElementById('zero');
+btnzero.addEventListener('click', zero);
+
+function zero() {
+  disChar.innerHTML += num[0];
+  return num[0];
+}
+
+const operator = ['+', '/', '*', "-", '.']
+
+const btnmul = document.getElementById('mul');
+btnmul.addEventListener('click', mul)
+
+function mul () {
+  if (disResult.textContent !== ' ') {
+    let newCalc = disResult.textContent;
+    disChar.innerHTML = ' ';
+    disChar.innerHTML += newCalc + operator[2];
+    disResult.textContent = ' ';
+    
+  }else {
+    disChar.innerHTML += operator[2];
+  }
+  
+}
+
+const btnadd = document.getElementById('add');
+btnadd.addEventListener('click', add);
+
+function add () {
+  if (disResult.textContent === ' ') {
+    disChar.innerHTML += operator [0];
+    // let newCalc = disResult.textContent;
+    // disChar.innerHTML = ' ';
+    // disChar.innerHTML += newCalc + operator[0];
+    // disResult.textContent = ' ';
```

Comment:
Leaving commented blocks/lines of code in your scripts makes your code messy and difficult to read. The best thing is to have them removed completely. 

However, If you want to make reference to some of these commented lines of code in the future, you should have a look at [this article ](https://kentcdodds.com/blog/please-dont-commit-commented-out-code)on how to handle that.